runtime/rasdump/javadump.cpp

```diff
@@ -3108,6 +3108,12 @@ JavaCoreDumpWriter::writeSharedClassSectionTopLayerStatsHelper(J9SharedClassJava
 	);
 	_OutputStream.writeInteger(javacoreData->debugAreaLocalVariableTableBytes, "%zu");
 	_OutputStream.writeCharacters("\n");
+
+	_OutputStream.writeCharacters(
+			"\n2SCLTEXTONA            OS Number Attached (nattach)              = "
```

Comment:
The leading `\n` is unwanted here.

Reply:
(unless the previous call to `writeCharacters("\n")` is removed)

Reply:
Removing the `writeCharacters("\n")` seems the better approach, updated with that change.

Reply:
I'm not a fan of the practice of including the newline at the beginning of the next line, but (half of) this method already does so.